index.html

```diff
@@ -301,66 +297,83 @@ <h3>Timeline</h3>
-                <td>Q2 2021</td>
-                <td>Q2 2021</td>
+                <td>Q4 2021</td>
+                <td>Q2 2022</td>
+                <td>Q1 2023</td>
+                <td>Q2 2023</td>
               </tr>
               <tr>
                 <th>Media Source Extensions</th>
-                <td>Q1 2020</td>
-                <td>Q4 2020</td>
-                <td>Q2 2021</td>
                 <td>Q2 2021</td>
+                <td>Q1 2022</td>
+                <td>Q4 2022</td>
+                <td>Q1 2023</td>
               </tr>
               <tr>
                 <th>Encrypted Media Extensions</th>
-                <td>Q1 2020</td>
-                <td>Q4 2020</td>
-                <td>Q2 2021</td>
-                <td>Q2 2021</td>
+                <td>Q4 2021</td>
+                <td>Q3 2022</td>
+                <td>Q1 2023</td>
+                <td>Q2 2023</td>
+              </tr>
+              <tr>
+                <th>WebCodecs</th>
+                <td>Q1 2021</td>
+                <td>Q2 2022</td>
+                <td>Q1 2023</td>
+                <td>Q2 2023</td>
               </tr>
             </tbody>
           </table>
           <p><em>
             Note: The actual production of some of the deliverables may
               follow a different timeline. Schedule changes will be documented on
-              the <span class="todo"><a href="">group home page</a></span>.
+              the <a href="https://www.w3.org/media-wg/">group home page</a>.
           </em></p>
         </section>
       </section>
 
+      <section id="success-criteria">
+        <h2>Success Criteria</h2>
+        <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of each of feature defined in the specification.</p>
+        <p>Each specification should contain a section detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+        <p>Each specification should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximising accessibility in implementations. The latest versions of the <a href="https://www.w3.org/TR/media-accessibility-reqs/">Media Accessibility User Requirements</a> and <a href="https://w3c.github.io/apa/fast/">Framework for Accessible Specification of Technologies (FAST)</a> documents notably provide media related advice to ensure that specifications developed by the Working Group meet the needs of users with disabilities.</p>
+        <p>To promote interoperability, To promote interoperability, all changes made to specifications should have <a href="https://www.w3.org/2019/02/testing-policy.html">tests</a>.</p>
```

Comment:
One "To promote interoperability," can be removed.

Reply:
Thanks!